### ISSUE TITLE
Admob policy violation: Modified ad behavior

### ISSUE THREAD

**Issue by HeberthFeres** [admob-plus/admob-plus#452]
Describe the bug

I have just received an admob policy violation & ad serving restricted due to:

Modified ad behavior
Publishers are not permitted to alter the behavior or targeting of Google ads. This includes implementing the AdSense ad code in a "floating box script," or altering the ad-targeting using hidden keywords or IFRAMES.

However, I have not changed any code for this to happen.

**Comment by ratson** [comment]
@HeberthFeres It says `AdSense`, not AdMob, could it be caused by your AdSense code?
It would be helpful if you could share how the ad code is called and which API are used.

**Comment by HeberthFeres** [comment]
I don't use AdSense in my app and it's not a new app. I have kept it for over 4 years.

Package:
admob-plus-cordova

`admob.start().then(async ( ) => { ... })`

`window.admob.BannerAd({
                                adUnitId: ADMOB_ID.banner,
                                position: 'bottom',  
                                size: _AD_SIZE_BANNER.SMART_BANNER
                            });
...
admob.banner.show();                            
`

`admob.InterstitialAd({
                                adUnitId: ADMOB_ID.interstitial
                            });
...
admob.interstitial.show();`

**Comment by HeberthFeres** [comment]
After several appeals. After changing several things in hopes of solving the problem. The solution was to remove the banner from the application. So they approved. Then I resent it with the banner again. I hope they don't limit it again.

**Comment by merbin2012** [comment]
Yes, I have also received the same notification from google (Android & iOS), but after updating this plugin they approved. But some of the other applications also use the same version but have not yet received this notification for those apps.

**Comment by EYALIN** [comment]
@ratson you can close it
